redirect.install

```diff
@@ -148,3 +148,53 @@ function redirect_update_8103() {
   }
   return $message;
 }
+
+/**
+ * Add redirect_error table.
+ */
+function redirect_update_8003() {
+  $schema = array(
+    'description' => 'Stores information on redirect errors.',
+    'fields' => array(
+      'reid' => array(
+        'type' => 'serial',
+        'not null' => TRUE,
+        'description' => 'Primary Key: Unique redirect error ID.',
+      ),
+      'source' => array(
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => TRUE,
+        'description' => 'The source path where the error happened.',
+      ),
+      'uid' => array(
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+        'description' => 'The {users}.uid of the user who created the error.',
+      ),
+      'language' => array(
+        'description' => 'The language this error is for; if blank, language is undefined',
+        'type' => 'varchar',
+        'length' => 12,
+        'not null' => TRUE,
+        'default' => 'und',
+      ),
+      'timestamp' => array(
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+        'description' => 'The timestamp of when the error was created.',
+      ),
+    ),
+    'primary key' => array('reid'),
+    'indexes' => array(
+      'source_language' => array('source', 'language'),
+    ),
+  );
+  db_create_table('redirect_error', $schema);
```

Comment:
We're not storing "redirect errors". We store 404 logs. So maybe redirect_404 as table name and similar updated descriptions.